recipe/meta.yaml

```diff
@@ -12,16 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   noarch: python
 
 requirements:
   host:
-    - python
+    - python >=3.6
```

Comment:
Is this the minimum supported version of https://github.com/taynaud/python-louvain? I *think* it will work with python versions less than 3.6 too?

Reply:
They do not have a minimum python version; however, this is the minimum conda-forge version at the moment.